src/configUtils.ts

```diff
@@ -21,8 +21,16 @@ export function buildTSFixIds(): string[] {
 
 export function getEditorConfig(configUri: Uri): FormatCodeSettings {
   const indentSize = workspace.getConfiguration("editor", configUri).get("tabSize", 2);
-  
+
   return {
     indentSize,
   };
 }
+
+export function getCoffeescripConfig() {
```

Comment:
Just missing the `t` in `getCoffeescripConfig`.

Reply:
derp